src/main/connector/system/LinuxProcess/LinuxProcess.yaml

```diff
@@ -0,0 +1,53 @@
+extends:
+- ../System/System
+connector:
+  displayName: LinuxProcess
+  platforms: Any platform with LinuxOS
+  reliesOn: Linux OsCommands
+  information: Gives process specific information and metrics
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+    - linux
+    criteria:
+    - type: commandLine
+      commandLine: /usr/bin/uname -o || /bin/uname -o
+      expectedResult: GNU/Linux
+      errorMessage: Not a valid Linux host.
+    tags: [ system, linux ]
+monitors:
+  process:
+    simple:
+      sources:
+        process:
+          type: commandLine
+          commandLine: ps aux | grep '${var::processNames}'
```

Comment:
Doing a `grep` with `${var::processNames}` is dangerous, as the `processNames` variable could contain special chars interpreted by the shell (backslashes or quotes). Also, the grep is performed simply on the output of the `ps aux` command, and therefore could match on other fields than just the process name (PID, PPID, CPU, etc.)

IMO we should use several `keepOnlyMatchingLines` compute steps to properly filter the output of `ps aux`, on process names, user ID, or command line, with 3 variables:

* `${var::processNameRegex}`
* `${var::processUserRegex}`
* `${var::processCommandRegex}`